docs/conf.py

```diff
@@ -37,6 +38,7 @@ def noop_decorator(*args, **kwargs):
     sys.modules["cupy"].ndarray = type("ndarray", (object,), {})
 if not hasattr(sys.modules["pyhdk"], "PyDbEngine"):
     sys.modules["pyhdk"].PyDbEngine = type("PyDbEngine", (object,), {})
+    sys.modules["pyhdk"].__version__ = "0.6.0"
```

Comment:
Shouldn't this variable be defined in `pyhdk` itself?

Reply:
I put it under its own attribute check. Is that what you were referring to?

Reply:
Ah, these are fake modules, I get it.

Reply:
Is it supposed to keep the minimum supported version of the hdk here? Synchronizing with the version in `env_hdk.yml`?

Reply:
> Is it supposed to keep the minimum supported version of the hdk here? Synchronizing with the version in `env_hdk.yml`?

@alexbaden in that case let's update to `0.7`.

Reply:
These are fake attributes. Do we need an up to date value here?

Reply:
Documentation is built in an environment where hdk does not installed (only fake module). If we do not update this variable, then not all hdk's parameters will be specified in https://modin.readthedocs.io/en/latest/flow/modin/config.html.

Reply:
Got it. Then, to avoid updating this attribute each time, when a new parameter is added, I would suggest:
```python
sys.modules["pyhdk"].__version__ = "999"
```

Reply:
It is also desirable to leave a comment why it is done this way.